recipes/instaseis/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set version = "1.0.0" %}
+{% set sha256 = "7fece415cca6c70e575531a463c0d5cb6d30f59740d7a113a02d584775a8b0c1" %}
+
+package:
+  name: instaseis
+  version: {{ version }}
+
+
+source:
+  fn: instaseis-{{ version }}.zip
+  url: https://pypi.io/packages/source/i/instaseis/instaseis-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - gcc
+  run:
+    - python
+    - libgfortran
+    - numpy
+    - h5py
+    - obspy
+    - future
+    - tornado
+    - requests
+    - jsonschema
+    - pyqt
+    - pyqtgraph
+
+test:
+  imports:
+    - instaseis
+  commands:
+    - python -m instaseis.tests
+  requires:
+    - click
+    - netcdf4
+    - responses
+    - pytest
+    - pytest-xdist
+    - mock  # [py27]
+
+about:
+  home: http://instaseis.net
+  license: LGPL-3.0 (for non-commercial/academic use)
+  summary: 'Instant Global Broadband Seismograms Based on a Waveform Database'
+  description: |
+    Instaseis calculates broadband seismograms from Green's function databases
+    generated with AxiSEM and allows for near instantaneous (on the order of
+    milliseconds) extraction of seismograms. Using the 2.5D axisymmetric
+    spectral element method, the generation of these databases, based on
+    reciprocity of the Green's functions, is very efficient and is approximately
+    half as expensive as a single AxiSEM forward run. Thus this enables the
+    computation of full databases at half the cost of the computation of
+    seismograms for a single source in the previous scheme and hence allows to
+    compute databases at the highest frequencies globally observed. By storing
+    the basis coefficients of the numerical scheme (Lagrange polynomials),
+    the Green's functions are 4th order accurate in space and the spatial
+    discretization respects discontinuities in the velocity model exactly. On
+    top, AxiSEM allows to include 2D structure in the source receiver plane and
+    readily includes other planets such as Mars.
+  doc_url: http://instaseis.net
+  dev_url: https://github.com/krischer/instaseis
+
+extra:
+  recipe-maintainers:
+    - krischer
+    - martinvandriel
+    - sstaehler
```

Comment:
Are both of you ok with being added as maintainers of `instaseis` at [conda-forge]( http://conda-forge.github.io/ ), @martinvandriel and @sstaehler?